test/kwok/kwok.test.sh

```diff
@@ -54,6 +54,32 @@ function test_node_ready() {
   fi
 }
 
+# Check pod status
+function test_check_pod_status() {
+  for row in $(kubectl get po -ojson | jq -r '.items[] | @base64'); do
+      _jq() {
+      echo ${row} | base64 --decode | jq -r ${1}
+      }
```

Comment:
```suggestion
  _jq() {
    echo ${row} | base64 --decode | jq -r ${1}
  }
  for row in $(kubectl get po -ojson | jq -r '.items[] | @base64'); do
```

Reply:
Applied the changes [here](https://github.com/kubernetes-sigs/kwok/compare/6937431a58b0a2149d9fe50882102895d9b55002..4849934ad12e13eb52cffd10cc563bf79e04f519).